agent/php_internal_instrument.c

```diff
@@ -796,6 +800,35 @@ static void nr_php_instrument_datastore_operation_call(
   }
 }
 
+/* 
+ * Like nr_php_instrument_datastore_operation_call() but does not
+ * call the underlying internal function. This allows multiple
+ * datastore metrics to be creating during a single underlying
+ * PHP call.
+ */
+static void nr_php_instrument_datastore_operation(
+    nr_datastore_t datastore,
+    const char* operation,
+    nr_datastore_instance_t* instance,
+    bool instance_only) {
+  nr_segment_t* segment = NULL;
+  nr_segment_datastore_params_t params = {
+      .datastore = {
+          .type = datastore,
+      },
+      .operation = nr_strdup(operation),
```

Comment:
Since this function is used only for memcached_multi_connect, this strdup and later free could be avoided because memcached_multi_connect doesn't have any operation associated with it.

Reply:
Because `operation` is `NULL` in this case, the impact is minimal. Because there is a potential use case for this function with an actual defined `operation`, I would rather leave it as is.